applications/coreutils/cp.cpp

```diff
@@ -0,0 +1,65 @@
+#include <libsystem/io/Filesystem.h>
+#include <libsystem/io/Stream.h>
+
+// implementation of cp open a input stream from file to copied and create a new file and open it as output stream and
+// paste all the content into it.
+// Add more options
+// - Create new file
+// - append to already existing file
+// - only copy if file already exists
+// - only logs if used with option -v
+
+int main(int argc, char **argv)
+{
+    if (argc == 3)
+    {
+        // open input stream
```

Comment:
Please avoid comment that paraphrases the code. Comments should answer the why and the code the how :)

Reply:
removed the comments :)